src/components/ResearchGroupCard/style.scss

```diff
@@ -1,23 +1,120 @@
-.App {
-  text-align: center;
+$card-header-font: 24px;
+$card-header-height: 64px;
+
+.card{
+    text-align: center;
+    font-family: 'Futura', sans-serif;
+    background-color: #fafafa;
+    border-style: solid;
+    border-color: #cecece;
```

Comment:
For all colors, make sure to save them as variables into . So you could save this as `$grey-med`, and also have colors for `$grey-light` and `$grey-dark`. Actually, I see that there already exists `$grey1/2/3` colors in the `src/styles` folder, so maybe use those or modify those. Make sure to import from those colors. 

This is in contrast to the variables you have above which are `$card-header-font` and `$card-header-height`, which are specific to cards so you can keep them locally in this file. Colors, fonts, etc. are global which is why you want to put them in the global styles.

Same for fonts, store or retrieve them from the global `src/styles/fonts`. That's why we're using sass, to keep things consistent and easy to share via variables. 

Finally, for borders, you can actually use shorthand eg:
`border: <border-width> <solid/dashed/etc> <color>`
would be
`border: 1px solid $grey-med`

Reply:
That's great programming practice! Thank you! I've fixed it now.